### PR TITLE
Update codeceptjs

### PR DESCRIPTION
**Pull request by frederikprijck**
We are using an incorrect docker image that hasn't been updated for 3 years: https://hub.docker.com/r/codeception/codeceptjs

Instead, we should be using https://hub.docker.com/r/codeceptjs/codeceptjs instead.

### CODE REVIEW

**Comment by frederikprijck** [comment]
Marking this as draft as the latest codeceptjs/codeceptjs uses a new major version of codeceptjs which requires changes. We have applied these changes on the chore/update-deps branch for our tests.

Once all samples have been updated and CI succeeds, we will merge the quickstart test changes to master and update this PR.

File: .circleci/config.yml [hunk 1]
```diff
@@ -57,8 +57,8 @@ commands:
           name: Run tests
           working_directory: test
           command: |
-            docker create --network host --name tester codeception/codeceptjs codeceptjs run-multiple --all --steps --verbose
-            docker cp $(pwd)/lock_login_spa_test.js tester:/tests/lock_login_test.js
+            docker create --network host --name tester codeceptjs/codeceptjs codeceptjs run-multiple --all --steps --verbose
+            docker cp $(pwd)/lock_login_test.js tester:/tests/lock_login_test.js
```
Comment (jimmyjames) [inline .circleci/config.yml]:
```suggestion
            docker cp $(pwd)/lock_login_spa_test.js tester:/tests/lock_login_spa_test.js
```

**Review by jimmyjames — CHANGES_REQUESTED** [review]
We should be running `lock_login_spa_test`, not `log_login_test`, as the `lock_login_test` does not click the profile button before waiting for the logout button. Making this change _should_ fix the test, at least it did locally for me.

File: .circleci/config.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ jobs:
       image: << pipeline.parameters.machine_image >>
     steps:
       - checkout
-      - run: git clone https://github.com/auth0-samples/spa-quickstarts-tests test
+      - run: git clone --branch chore/update-deps https://github.com/auth0-samples/spa-quickstarts-tests test
```
Comment (jimmyjames) [inline .circleci/config.yml]: Does this branch still exist? CI failing because it doesn't exist, same as locally. Did the change get merged to main?

Comment (jimmyjames) [inline .circleci/config.yml]:
```suggestion
      - run: git clone https://github.com/auth0-samples/spa-quickstarts-tests test
```

Comment (frederikprijck) [reply .circleci/config.yml]: Yes it did, sorry about that.

**Pull request closed without merging**